apps/aecore/lib/aecore/miner/worker.ex

```diff
@@ -241,11 +226,14 @@ defmodule Aecore.Miner.Worker do
                       top_block.header.height + 1 +
                       Application.get_env(:aecore, :tx_data)[:lock_time_coinbase]) |
                    valid_txs_by_fee]
-      root_hash = BlockValidation.calculate_root_hash(valid_txs)
 
       new_chain_state = ChainState.calculate_and_validate_chain_state!(valid_txs, chain_state, top_block.header.height + 1)
       chain_state_hash = ChainState.calculate_chain_state_hash(new_chain_state)
 
+      #valid_txs_with_oracle_registration_txs = oracle_registration_txs ++ valid_txs
+
+      root_hash = BlockValidation.calculate_root_hash(valid_txs)
```

Comment:
doesn't this include the `oracle_registration_txs` ?

Reply:
It does, I've forgotten to remove the comment